### PR TITLE
feat(version-info): update to v5.20.0

### PR DESCRIPTION
**Pull request by nikku**
Proposing the following what's new communication:

image

### CODE REVIEW

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -45,15 +45,15 @@ export function ReleaseInfo(props) {
       <ul className="dashed">
         <li>
           <h4>New form fields</h4>
-          The form editor now supports dynamic lists and tables. This allows you to work with static and dynamic lists of data in complex forms.
+          The form editor now supports a HTML component that allows you to embed simple custom widgets.
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:48]: Nitpick: "a**n** HTML component"

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -45,15 +45,15 @@ export function ReleaseInfo(props) {
...
         </li>
         <li>
-          <h4>Simplified compensation modeling</h4>
-          The BPMN editor ships with improved compensation handling. Turn any activity into a compensation activity by connecting it to a  compensation boundary event.
+          <h4>Improved properties and FEEL editing</h4>
+          We improved various aspects of our properties and FEEL editing experience and made FEEL validation errors more helpful.
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:52]: What properties?

**Comment by lmbateman** [comment]
This looks good to me! A couple of very minor suggestions:

1. "a HTML component" --> "an HTML component"
2. "Remember recently opened files" -- the other headers are noun phrases, and this is a verb phrase. Maybe "List of recently-opened files" or just "Recently-opened files"? (Also, I think "recently-opened" should have a hyphen because it's a modifier.)
3. "re-open" --> "reopen"
4. For "Improved properties and FEEL editing", is there somewhere I can go for a full list of improvements? Not strictly necessary, but people might be curious. I think the others are fairly self-explanatory.

**Review by lmbateman — COMMENTED** [review]
This looks good to me! A couple of very minor suggestions:

1. "a HTML component" --> "an HTML component"
2. "Remember recently opened files" -- the other headers are noun phrases, and this is a verb phrase. Maybe "List of recently-opened files" or just "Recently-opened files"? (Also, I think "recently-opened" should have a hyphen because it's a modifier.)
3. "re-open" --> "reopen"
4. For "Improved properties and FEEL editing", is there somewhere I can go for a full list of improvements? Not strictly necessary, but people might be curious. I think the others are fairly self-explanatory.

**Comment by nikku** [comment]
Thanks for your feedback. I think I addressed most of it with https://github.com/camunda/camunda-modeler/pull/4101/commits/697e523ec33af0f1a29b7ee7011fd4e577edceff.

**Pull request merged**